infrastructure/nomad/playbooks/templates/jobs/mev-commit-funder.nomad.j2

```diff
@@ -68,19 +68,21 @@ job "{{ job.name }}" {
           mkdir -p "${KEYSTORE_PATH}" > /dev/null 2>&1
           {{- with secret "secret/data/mev-commit" }}
           KEYSTORE_FILE="${KEYSTORE_PATH}/${KEYSTORE_FILENAME}"
-          echo '{{ .Data.data.{% endraw %}{{ job.target.artifacts | selectattr('keystores', 'defined') | map(attribute='keystores') | first | list | first }}{% raw %} }}' > "${KEYSTORE_FILE}"
+          echo '{{ .Data.data.contract_deployer_keystore }}' > "${KEYSTORE_FILE}"
           {{ end }}
 
           {{- range nomadService "{% endraw %}{{ job.target.name }}{% raw %}" }}
+            {{- if contains "http" .Tags }}
+          ADDRESS=$(curl -sk https://{{ .Address }}:{{ .Port }}/v1/debug/topology | jq -r '.topology.self["Ethereum Address"]')
+            {{- end }}
             {{- range nomadService "mev-commit-geth-bootnode1" }}
               {{- if contains "http" .Tags }}
-          ADDRESS=$(cat "${KEYSTORE_FILE}" | jq -r '.address')
           cast send \
             --keystore "${KEYSTORE_FILE}" \
             --password "${KEYSTORE_PASSWORD}" \
             --priority-gas-price 2000000000 \
             --gas-price 5000000000 \
-            --value 1000ether \
+            --value 10ether \
```

Comment:
The mev-commit contract deployer can no longer send unaccounted for ether to various accounts. All usable eth on the mev-commit chain (excluding balance given to allocator contract) must have an equivalent amount locked in the L1 gateway contract

Reply:
Funders are only used in testnet/devnet.

Reply:
We can keep the current code to move quickly, but ideally we eventually have constant config and test bidders/providers bridge their eth from mock l1

Reply:
Fixed.